src/app/home/home.component.html

```diff
@@ -82,50 +83,60 @@ <h2>About this Site</h2>
           This site was created to provide a unified space where land and water planning activities
           can be shared to ensure transparency of these processes in the province. It is a portal to
           learn about land and water planning projects in B.C., access supporting documents and reports,
-           and submit comments and feedback to planning teams involved in active planning projects.
+          and submit comments and feedback to planning teams involved in active planning projects.
         </p>
         <p>
-          Land and water planning in B.C. happens at different scales. Planning can occur at a strategic
-           level where major land use zones guide conservation and economic outcomes, and at a tactical
-            level to provide more detailed guidance for specific activities and objectives. Each planning
-             process is designed to address the issues and challenges within the plan area, mitigate the
-              effects of climate change on communities, advance reconciliation, and provide clarity on
-              how the land base will be managed for all economic, cultural, and conservation objectives.
-               The Province is adopting a more integrated approach to planning that will result in
-               coordinated processes to achieve various planning objectives. This site provides access
-                to information on all active land use planning, forest landscape planning, and water
-                 planning projects.
+          <a href="https://www2.gov.bc.ca/assets/gov/farming-natural-resources-and-industry/forestry/forest-landscape-plans/land-and-water-planning.pdf"
+            target="_blank" rel="noopener noreferrer"
+            aria-label="Land and water planning in B.C. (PDF, 2.6 MB, opens in new tab)">
+            Land and water planning in B.C.
+            <i class="material-icons" aria-hidden="true">picture_as_pdf</i>
+            <i class="material-icons" aria-hidden="true">open_in_new</i>
```

Comment:
I think we decided against using these icons on the site. I was also an advocate of them. I won't block the PR, but it might be worth checking with Steve.